_rules/text-spacing-not-important-b8bb68.md

```diff
@@ -0,0 +1,288 @@
+---
+id: b8bb68
+name: Text spacing in `style` attributes is not `!important`
+rule_type: atomic
+description: |
+  This rule checks that text spacing set in the `style` attribute is below permissible minimum value or not set to `!important`
+accessibility_requirements:
+  wcag21:1.4.12: # Text Spacing (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Jey Nandakumar
+---
+
```

Comment:
Given that the 3 expectations are completely independent, I am not sure we gain much by keeping this a single rule. This could be turned into 3 small rules instead. I think that small rules are more to the point and easier to "swallow" and understand.

This actually goes well with moving !important to applicability:

> Applicability:
> This rule applies to any HTML element that is visible and has a declared value of the word-spacing property that is specified by the style attribute with the important flag set

> Expectation:
> The computed value of the word-spacing property is at least …

Reply:
I think it may be specific enough and small enough once "!important" is part of the applicability to remain as one rule about text spacing styles.